microsoft/electionguard-python
# 🕸️ #669 Add verification URL to CLI

[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue
Fixes #669 

### Description
Adds the ability to pass a URL parameter to electionguard setup, also that url will get passed through to the election package when you import ballots.

### Testing
1. Run `make eg-setup-simple-election` for a setup election without a URL

Expected: the resulting context shouldn't have any extended data.

2. Run `poetry run eg setup --guardian-count=2 --quorum=2 --manifest=data/election_manifest_simple.json  --package-dir=../data/out/public_encryption_package --keys-dir=../data/out/test_data_private_guardian_data --url=https://electionguard.vote` to see it work with a url.  

Expected: The resulting context should have extended data with a url in it.
